dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -1719,7 +1720,18 @@ public PhaseOneResult process(CompilationUnitTree root, UnderlyingAST underlying
          *
          * @param tree the newly created Tree
          */
-        public void handleArtificialTree(Tree tree) {}
+        public final void handleArtificialTree(Tree tree) {
```

Comment:
Options:
1. Consider lambda
2. move this to CFCFGBuilder, add comment to encourage using this one-argument version